boilerplate-swiftui-bloc/boilerplate_swiftui_blocApp.swift

```diff
@@ -6,11 +6,16 @@
 //
 
 import SwiftUI
+import Repository
 
 @main
 struct boilerplate_swiftui_blocApp: App {
     let router = NavigationRouter(routes: .all)
     
+    init() {
+        RepositoryConstants.setup(with: Bundle.main)
```

Comment:
Configs should be a part of Repository, we don need one more singleton for RepositoryConfigs